modules/rosa-policy-shared-responsibility.adoc

```diff
@@ -370,8 +369,8 @@ managed KMS key that Amazon EBS provides.
 - Provide the ability for the customer to create a public or private Amazon S3 image registry to protect their container
 images from unauthorized user access.
 
-|- Provision Amazon EBS volumes.
```

Comment:
Revoked. As the service name is associated with shared resource volumes.

Reply:
Updated, as per the https://github.com/openshift/openshift-docs/pull/59650#discussion_r1290144425